lib/Dialect/LLVMIR/IR/LLVMDialect.cpp

```diff
@@ -1218,6 +1218,8 @@ LLVMDialect::LLVMDialect(MLIRContext *context)
   impl->doubleTy = LLVMType::get(context, llvm::Type::getDoubleTy(llvmContext));
   impl->floatTy = LLVMType::get(context, llvm::Type::getFloatTy(llvmContext));
   impl->halfTy = LLVMType::get(context, llvm::Type::getHalfTy(llvmContext));
+  impl->fp128Ty = LLVMType::get(context, llvm::Type::getFP128Ty(llvmContext));
+  impl->x86_fp80Ty = LLVMType::get(context, llvm::Type::getX86_FP80Ty(llvmContext));
```

Comment:
Can you wrap this around 80 characters? It is failing clang format.

Reply:
Oops. Fixed.